authgear/authgear-server
# Support OAuth in SignupFlow and LoginFlow

ref #3273 

## How to test
1. Use the original redirect URI as `redirect_uri` (e.g. http://localhost:3100/sso/callback/google)
2. Copy the `oauth_authorization_url` and visit it in the browser.
3. Open the browser inspector, turn on persistent log.
4. Finish the authorization in the provider website.
5. Being redirected from the provider to /soo/callback/:alias and see an error page. This is expected.
6. From the network log, find out the 302 response and copy the authorization code.
7. Send the authorization code to the Authentication Flow to continue.
